OracleFMWInfrastructure/samples/12213-domain-home-in-image/Dockerfile

```diff
@@ -67,12 +67,13 @@ COPY container-scripts/* /u01/oracle/container-scripts/
 #Create directory where domain will be written to
 USER root
 RUN chmod +x /u01/oracle/container-scripts/* && \
-    mkdir -p ${PROPERTIES_FILE_DIR} && \
+    mkdir -p +xwr ${PROPERTIES_FILE_DIR} && \
     chown -R oracle:oracle ${PROPERTIES_FILE_DIR} && \
     mkdir -p $DOMAIN_HOME && \
-    chown -R oracle:oracle $DOMAIN_HOME && \
-    chmod -R a+xwr $DOMAIN_HOME
+    chown -R oracle:oracle $DOMAIN_HOME/.. && \
+    chmod -R a+xwr $DOMAIN_HOME/..
```

Comment:
I don't think this was the intention of the writer (of the sample).  I think the intention was to change the permissions of the domain home directory.  Why is it necessary to change the ownership of its parent directory?

Reply:
The sample will not work without changing the parent folder of the domain, the create domain handling in WLST fails with file permission errors.

Also, the `domain-home-in-image-wdt` sample which is switching back to `oracle` user from `root addresses the file permissions from the domain parent folder level. These changes are as such consistent between these domain in image samples.

One additional comment from me is that since `oracle` is the user that will run the image, having the parent folder ownership is quite valid IMHO since the `oracle` user can create/use additional folders for `user_projects` like logs or even additional domain folders can be created.